cabal-testsuite/PackageTests/ReplBuildDepends/cabal.allow-newer.out

```diff
@@ -0,0 +1,2 @@
+# cabal clean
+# cabal repl
```

Comment:
Are `cabal.allow-newer.out` and `cabal.allow-older.out` still needed?

Reply:
It seems not. Good catch!

Reply:
Well, yes they were supposed to… I just realise I gave the same name to each individual test. Fixed now. But yes the `*.out` files are identical.